public/background.js

```diff
@@ -1,17 +1,19 @@
-chrome.runtime.onInstalled.addListener(function (details) {
-  if (details.reason == "install") {
-    chrome.tabs.create({
-      url: "https://www.wikitree.com/wiki/Space:WikiTree_Browser_Extension",
-      active: true,
-    });
-    chrome.runtime.openOptionsPage();
-  } else if (details.reason == "update") {
-    // Use this to open the extension update page on update
-    /*
+if (chrome.runtime) {
+  chrome.runtime.onInstalled.addListener(function (details) {
+    if (details.reason == "install") {
+      chrome.tabs.create({
+        url: "https://www.wikitree.com/wiki/Space:WikiTree_Browser_Extension",
+        active: true,
+      });
+      chrome.runtime.openOptionsPage();
+    } else if (details.reason == "update") {
+      // Use this to open the extension update page on update
+      /*
     chrome.tabs.create({
       url: "https://www.wikitree.com/wiki/Space:WikiTree_Browser_Extension_Update",
       active: true,
     });
     */
```

Comment:
Should this still be commented out?

Reply:
Yes. That's for opening the update page when we have new features. We don't need it for small things like bug fixes.

Reply:
Got it. Should we also then make use of [onUpdateAvailable](https://developer.chrome.com/docs/extensions/reference/runtime/#event-onUpdateAvailable) or would that be overkill?

Reply:
I don't know.  What would you want to do with it?  Reload the extension if an update is available?  It could bother people.  When we set it  to open the update page on update, it's OK, because people are used to having pages opened by extensions when they open their browser (or I am at least).  If we reload it while people are working, they might be surprised by the update page opening up.  What do you think?

Reply:
I was thinking along the lines of a pop-up notice of an update being available.

Reply:
I think if you want to do that, then feel free, but I think that may also bother people - like we're nagging them to do something (even if it's just to click something). I don't know.  I'd like to hear more opinions.